adapt-and-extend/theming/theme-with-static-website-template.rst

```diff
@@ -0,0 +1,80 @@
+==========================================
+Create a theme based on a website template
+==========================================
+
+Introduction
+------------
+
+We have a static theme based on HTML/CSS and want to use it for Plone. The theme separates the frontend for anonymous users from the backend with default Plone UI for editors.
+
+
+Requirements
+++++++++++++
+
+We assume that you have a Plone site up and running. You are logged in as a Manager and can access the Control Panel.
+
+
+Getting started
+---------------
+
+TODO
+
+
+Create a theme package
+++++++++++++++++++++++
+
+If you already have a package you can skip this step. If you start from scratch you need to create a theme package for Plone. This manual shows how you can create a package.
+
+Link to bobtemplates...
+
+
+Download an existing theme
+++++++++++++++++++++++++++
+
+In our example we start by downloading an existing bootstrap-based theme from Start Bootstrap.
+
+http://startbootstrap.com/template-overviews/business-casual/
+
+
```

Comment:
We should make 'Start Bootstrap/ the link, much easier on the eye and for reading